frontend/www/js/omegaup/course/clarifications.ts

```diff
@@ -1,3 +1,37 @@
 import { OmegaUp } from '../omegaup';
+import { types } from '../api_types';
+import Vue from 'vue';
+import * as api from '../api';
+import * as ui from '../ui';
 
-OmegaUp.on('ready', () => {});
+import course_Clarications from '../components/course/Clarifications.vue';
```

Comment:
```suggestion
import course_Clarifications from '../components/course/Clarifications.vue';
```

jaja, me pasa lo mismo todo el tiempo.